packages/react/src/tooltip/MdTooltip.tsx

```diff
@@ -1,14 +1,24 @@
 import classnames from 'classnames';
 import React, { useState } from 'react';
 
-export interface MdTooltipProps {
-  content: React.ReactNode;
+export interface MdTooltipProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * Replaces previous content-prop for specifying the content of the tooltip.
+   * Content-prop is reserved as a standard HTML attribute on div-elements.
+   */
+  label: React.ReactNode;
```

Comment:
Burde denne hete tooltipContent ettersom label er litt vagt for å beskrive en ReactNode?